packages/ra-core/src/dataProvider/fetch.spec.ts

```diff
@@ -85,4 +92,40 @@ describe('createHeadersFromOptions', () => {
         );
         expect(headersWithoutMethod.get('Content-Type')).toBeNull();
     });
+
+    it('should not add a Content-Type header if there is no body', () => {
```

Comment:
This title reads like the one in L65. You should probably explicit the fact that it's for DELETE here